cli/src/utilities/printOutput.ts

```diff
@@ -95,7 +96,12 @@ export async function printOutput(
       const nativeWindUIComponents =
         cliResults.packages.find((p) => p.name === 'nativewindui').options.selectedComponents ?? [];
 
-      const finalComponents = Array.from(new Set([...nativeWindUIComponents, 'text']));
+      // we do this to account for older stored config e.g that has selectable text in it
+      const onlySupportedComponents = nativeWindUIComponents.filter((component) =>
```

Comment:
@dannyhw i'm not sure i follow what's happening here

Reply:
Just filtering to real component options. For example i removed the selectable text option so if someone passes it we need to remove it

Reply:
Like imagine someone manually writes out the selected components they want or they saved a config before the update and it includes selectable text or a typo for example

Reply:
Don't we already ignore whatever is passed in that isn't a validated option?

Reply:
well yes but theres no validation on stored config since it just directly outputs the cliResults, so we either add a check there or here

Reply:
Breaks it
![image](https://github.com/user-attachments/assets/a087c34e-9dc0-4f54-9f6b-785d9fb15653)

Reply:
so what if you pass in a flag like `--some-bullshit-component`, will that break it too?